tests/vmi_configuration_test.go

```diff
@@ -165,7 +165,16 @@ var _ = Describe("Configurations", func() {
 						kubev1.ResourceMemory: resource.MustParse("64M"),
 					},
 				}
-				vmi, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
+				// Retrying 5 times with its interval set to one second, then if you still succeeds
+				// in VMI creation, things must be going wrong.
+				for i := 0; i < 5; i++ {
+					vmi, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
```

Comment:
it possible situation when first creation passed, so when you will try to create VMI with the same name the second time, it will fail, but it not the same error that we expected.

Do we have some way to be sure, that cache already updated?

Reply:
That makes sense, will push a commit with randomising the name, while still deleting just after the creating error as is. Thanks.